src/Corax/Pipeline/WhitespaceTokenizer.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+
+namespace Corax.Pipeline
+{
+    public struct WhitespaceTokenizer : ITokenizer
+    {
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public int Tokenize(ReadOnlySpan<byte> source, ref Span<Token> tokens)
+        {
+            int i = 0;
+            int currentToken = 0;
+
+            while ( i < source.Length)
+            {
+                while (i < source.Length && source[i] == ' ')
```

Comment:
`\t`, `\n`, `\r` ? there are other whitespace chars, too.

Reply:
Yes, this `WhitespaceTokenizer` is a proof of concept to understand the pipelining aspects.